services/graphite/config.go

```diff
@@ -38,6 +38,7 @@ type Config struct {
 	BindAddress      string        `toml:"bind-address"`
 	Database         string        `toml:"database"`
 	Enabled          bool          `toml:"enabled"`
+	WriteSkip        bool          `toml:"write-skip"`
```

Comment:
This allows me to have the Graphite input perform all processing except actually writing to the database. It is tedious to constantly recompile the code, just to check input performance, in isolation. So the idea is a developer can set this flag, and try various other configs (mostly around batching and Goroutines). There have been some complaints about network-level performance and memory usage of Graphite, and I want to be able to test just the Graphite code.

Think of like the way we sometimes processing within the system, but drop the data before writing. It's most convenient if these performance-testing-related switches are configurable. They would not be documented in the sample config however.